modules/nw-egressnetworkpolicy-object.adoc

```diff
@@ -151,9 +151,9 @@ spec:
 
 As a cluster administrator, you can allow or deny egress traffic to nodes in your cluster by specifying a label using `nodeSelector`. Labels can be applied to one or more nodes. The following is an example with the `region=east` label:
 
-[source,yaml]
+[source,yaml,subs="attributes+"]
 ----
-apiVersion: v1
+apiVersion: {api}
```

Comment:
What's the value of {api}?

Reply:
Sorry, it renders as `apiVersion: k8s.ovn.org/v1` in the preview.